client/src/components/Search/SearchBox.scss

```diff
@@ -102,6 +102,13 @@ $search-box-height: 53px;
   &.collapsed::before {
     transform: rotate(0deg) translateY(0);
   }
+
+  &.search-box--no-label {
+    font-size: 0;
+    margin-right: 9px;
+    margin-top: 13px;
+    padding-right: 5px;
+  }
```

Comment:
Why font-size 0? The more standard approach to something like this would be to put the text into a `<span>` and add the `sr-only` css class to that span when we want to visually hide the text.

Reply:
Updated